scripts/Get-Tags.ps1

```diff
@@ -0,0 +1,32 @@
+param(
+    [string]$Branch='master',
+    [string]$ImageBuilderImageName='microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20171005132855',
+    [string]$RepoName
+)
+
+$ErrorActionPreference = 'Stop'
+$repoRoot = Split-Path -Path "$PSScriptRoot" -Parent
+
+if ([String]::IsNullOrWhiteSpace($RepoName))
+{
+    [Uri]$remoteUrl = New-Object -TypeName System.Uri -ArgumentList "https://github.com/dotnet/dotnet-docker-nightly.git"
+    if ([Uri]::TryCreate(((git config --get remote.origin.url) | Out-String), [UriKind]::Absolute, [ref]$remoteUrl))
+    {
+        $RepoName = [System.IO.Path]::GetFileNameWithoutExtension($remoteUrl.ToString())
+    }
+}
+
+if ([String]::IsNullOrWhiteSpace($RepoName))
```

Comment:
I think this case should be an error.  Having it default to `dotnet-docker-nightly` defeats the reason I asked for it to be calculated - so the script is identical between the repos.